docs/root/modules/duo/schema.md

```diff
@@ -0,0 +1,178 @@
+| push_enabled | Legacy parameter; no effect if specified and always returns false. |
+| sms_enabled | Legacy parameter; no effect if specified and always returns false |
+| status | The group's authentication status. May be one of: "Active", "Bypass", "Disabled" |
+| voice_enabled | Legacy parameter; no effect if specified and always returns false |
+
+#### Relationships
+
+- An DuoApiHost contains DuoGroups
+
+    ```
+    (DuoApiHost)-[RESOURCE]->(DuoGroup)
+    ```
+
+- A DuoUser is part of multiple DuoGroups.
+
+    ```
+    (DuoGroup)-[GROUP_DUO]->(DuoUser)
+    ```
+
+
+### DuoUser
+
+Represents a [user](https://duo.com/docs/adminapi#users) in Duo.
+
+| Field | Description |
+|-------|--------------|
+| firstseen| Timestamp of when a sync job first discovered this node  |
+| lastupdated |  Timestamp of the last time the node was updated |
+| **id** | The user_id |
+| alias1 | The user's username alias1. |
+| alias2 | The user's username alias2. |
+| alias3 | The user's username alias3. |
+| alias4 | The user's username alias4. |
+| aliases | Map of the user's username alias(es). Up to eight aliases may exist. |
+| created | The user's creation date as a UNIX timestamp. |
+| email | The user's email address. |
+| firstname | The user's given name. |
+| groups | List of groups to which this user belongs. See Retrieve Groups for response info. |
+| is_enrolled | Is true if the user has a phone, hardware token, U2F token, WebAuthn security key, or other WebAuthn method available for authentication. Otherwise, false. |
+| last_directory_sync | An integer indicating the last update to the user via directory sync as a Unix timestamp, or null if the user has never synced with an external directory or if the directory that originally created the user has been deleted from Duo. |
+| last_login | An integer indicating the last time this user logged in, as a Unix timestamp, or null if the user has not logged in. |
+| lastname | The user's surname. |
+| notes | Notes about this user. Viewable in the Duo Admin Panel. |
+| phones | A list of phones that this user can use. See Retrieve Phones for descriptions of the phone response values. |
+| realname | The user's real name (or full name). |
+| status | The user's status. One of: "active", "bypass", "disabled", "locked out", "pending deletion". |
+| tokens | A list of tokens that this user can use. A list of JSON strings |
+| u2f_tokens | A list of U2F tokens that this user can use. A list of JSON strings |
+| user_id | The user's ID. |
+| username | The user's username. |
+| webauthncredentials | A list of WebAuthn authenticators that this user can use. A list of JSON strings |
+
+#### Relationships
+
+- An DuoApiHost contains DuoUsers
+
+    ```
+    (DuoApiHost)-[RESOURCE]->(DuoUser)
+    ```
+
+- A DuoUser is part of multiple DuoGroups.
+
+    ```
+    (DuoGroup)-[GROUP_DUO]->(DuoUser)
```

Comment:
If we keep the GROUP_DUO name, we should point toward the group.

Else we should rename the rel label to something like "MEMBER_OF_DUO_GROUP" since this currently is pointing at the DuoUser

Reply:
renamed to MEMBER_OF_DUO_GROUP